src/dnaio/_core.pyx

```diff
@@ -239,10 +239,7 @@ cdef class SequenceRecord:
 
 
     def fastq_bytes_two_headers(self):
-        """
-        Return this record in FASTQ format as a bytes object where the header (after the @) is
-        repeated on the third line.
-        """
+        # Deprecated, use ``.fastq_bytes(two_headers=True)`` instead.
```

Comment:
should we issue a DeprecationWarning here? (Similar how to CPython tackles these).

Reply:
Sure. I doubt people are going to complain. Who wants bigger FASTQ data anyway?